Source/Spark.swift

```diff
@@ -51,8 +51,7 @@ public class Spark {
         let deviceService = DeviceService(authenticationStrategy: authenticationStrategy)
         let callManager = CallManager(authenticationStrategy: authenticationStrategy, deviceService: deviceService)
         let webSocketService = WebSocketService(authenticationStrategy: authenticationStrategy, callManager: callManager)
-        let applicationLifecycleObserver = ApplicationLifecycleObserver(webSocketService: webSocketService, callManager: callManager, deviceService: deviceService)
```

Comment:
The functions of the ApplicationLifecycleObserver were merged into the Phone. I've reviewed this, and the merge happened before the addition of the SparkBackgroundCallNotifications stuff that was added in a PR recently. That appears NOT to have been properly integrated when the two branches were merged, so that's bad. I'll talk to the team here and get it straightened out. Don't merge until then! :)